src/context-providers/builder/types.ts

```diff
@@ -52,6 +52,11 @@ export interface IBuilderState {
 // =============================================================================
 // ACTIONS
 // =============================================================================
+export interface IToggleSubmittingAction {
+    type: "toggle-submitting";
+    payload: boolean;
+}
+
```

Comment:
now that we have moved `isSubmitting` to side panel, this action is not needed anymore and should be removed